Service/ImportService.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+
+namespace EMS\CoreBundle\Service;
+
+
+use Doctrine\Bundle\DoctrineBundle\Registry;
+use EMS\CoreBundle\Elasticsearch\Bulker;
+use EMS\CoreBundle\Entity\ContentType;
+use EMS\CoreBundle\Helper\DocumentImporter;
+use Symfony\Component\Form\FormFactoryInterface;
+
+class ImportService
```

Comment:
This class acts as a factory for the DocumentImporter. However I think we only need this class.

Make an object for the initDocumentImporter parameters. then refactor the helper into this class and use the object as a context for every function call that is made.

Reply:
1e8555d195f4caf512272f5f4efed1125cc64771